[2025-03-27T19:10:03.962+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-27T19:10:03.980+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL.load scheduled__2025-03-27T19:05:00+00:00 [queued]>
[2025-03-27T19:10:03.990+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL.load scheduled__2025-03-27T19:05:00+00:00 [queued]>
[2025-03-27T19:10:03.991+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-27T19:10:04.005+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2025-03-27 19:05:00+00:00
[2025-03-27T19:10:04.015+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1387) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-27T19:10:04.014+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'load', 'scheduled__2025-03-27T19:05:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/HW6_ETL.py', '--cfg-path', '/tmp/tmp9f6bpf_y']
[2025-03-27T19:10:04.017+0000] {standard_task_runner.py:72} INFO - Started process 1400 to run task
[2025-03-27T19:10:04.017+0000] {standard_task_runner.py:105} INFO - Job 60: Subtask load
[2025-03-27T19:10:04.075+0000] {task_command.py:467} INFO - Running <TaskInstance: ETL.load scheduled__2025-03-27T19:05:00+00:00 [running]> on host 00cc07a75964
[2025-03-27T19:10:04.178+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T19:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-27T19:05:00+00:00'
[2025-03-27T19:10:04.180+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-27T19:10:04.195+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-27T19:10:04.196+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-03-27T19:10:04.197+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-27T19:10:04.198+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-27T19:10:05.141+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-27T19:10:05.737+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-27T19:10:06.093+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-27T19:10:06.256+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-27T19:10:06.257+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-27T19:10:06.270+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-27T19:10:06.271+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=load, run_id=scheduled__2025-03-27T19:05:00+00:00, execution_date=20250327T190500, start_date=20250327T191003, end_date=20250327T191006
[2025-03-27T19:10:06.310+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-27T19:10:06.325+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-27T19:10:06.327+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
